metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1755,7 +1751,22 @@ class MetalsLanguageServer(
         }.asJavaObject
 
       case ServerCommands.ShowTasty() =>
-        tastyHandler.executeShowTastyCommand(params).asJavaObject
+        val parsedArgs = for {
+          args <- Option(params.getArguments.asScala)
```

Comment:
You can use TextDocumentPositionParams and parse it from params, or you can wait for https://github.com/scalameta/metals/pull/3149#pullrequestreview-760642240 to get merged, might improve the UX here.

Though I wonder if it will parse correctly without the position and set it to null?

Reply:
👍🏼 on the https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams if possible. It will make implementing this way simpler for clients.

Reply:
`TextDocumentPositionParams` seems good, will use it!